forge-gui/res/cardsfolder/upcoming/pact_weapon.txt

```diff
@@ -0,0 +1,16 @@
+Name:Pact Weapon
+ManaCost:3 B
+Types:Artifact Equipment
+K:Equip:Discard<1/Card>
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | IsPresent$ Card.Self+AttachedTo Creature | AddKeyword$ You don't lose the game for having 0 or less life. | Description$ As long as CARDNAME is attached to a creature, you don't lose the game for having 0 or less life.
```

Comment:
Did you test this?

Shouldn't this affect `You`?

Reply:
It was tested, but I won the game before that came to pass. Will fix.